### ISSUE TITLE
[EXE] Logistic Regression as a Neural Network- Basic Deep Learning Algorithms

### ISSUE THREAD

**Issue by rakshaa2000** [gimseng/99-ML-Learning-Projects#105]
#### Learning Goals
[Learning goals, bulleted/numbered list is preferred]
[e.g. learn the concept and the use of train/validation/test dataset using scikit-learn ]
Learn the gradient descent algorithm using numpy and matplotlib.

### Exercise Statement
[Explain and describe what the exercise is]
[e.g. apply simple random-forest model to classify titanic survivability from titanic data ]
This exercise helps classify pictures as cat and non-cat with the help of neural networks with 2 layers.

### Prerequisites 
[Prerequisites, in terms of concepts or other exercises in this repo]
[e.g. random-forest model, stochastic gradient descent, exercise #32]
Must know the basics of logistic regression.

### Data source/summary:
[Provide a succinct summary of what the data is and where it is from] 
[e.g. This involves covid19 fatality dataset from John Hopkin's website (links..) ]
This is the solution to the Neural Networks Exercise in Coursera. It is a custom dataset.

### (Optional) Suggest/Propose Solutions
[e.g. I have the solution using PyTorch, will be happy to create pull request to include the exercise statement/solution]
[e.g. I think chapter 3 of A. Geron's textbook works out the solution for this exercise]
[e.g. fast.ai's chapter 5 has the perfect solution for this]
I have the solution notebook for this exercise. I will be happy to create pull request to include the exercise.

### (Optional) Further Links/Credits to Relevant Resources:
[e.g. This exercise and solution's proposal came from a lab session from DL2020]
Found this exercise [here](https://www.coursera.org/learn/neural-networks-deep-learning/ungradedLab/63Vsy/logistic-regression-with-a-neural-network-mindset)

**Comment by AjayKhalsa** [comment]
@rakshaa2000 can you use the cats vs dogs dataset instead, because that would be great.

**Comment by rakshaa2000** [comment]
Sure, I'll work on it ✌

**Comment by darpan-2001** [comment]
@rakshaa2000 can I work on this issue now?
